German.lang.bak

```diff
@@ -206,98 +206,98 @@ UPDATES: Vafels (05/12/2018), Vafels (26/03/2019), Vafels (19/05/2019), MangoMar
 198=Griff
 199=Schalldämpfer
 200=Normale Munition
-201=3. Munition geschweift
-202=Geschweifte Munition
+201=Leuchtspurmunition (Jeder 3. Schuss)
+202=Leuchtspurmunition
 203=Verlassen
 204=Team wechseln
 205=Beitritt//Trupp beitreten
-206=Arsenal//Arsenal ändern
-207=Anpassen//Charakter anpassen (Rüstung etc pp)
-208=Möchtest du @1 für @2 Punkte freizuschalten?
+206=Ausrüstung//Ausrüstung ändern
+207=Charakter anpassen //Charakter anpassen (Rüstung etc pp)
+208=Möchtest du die Klasse @1 für @2 Punkte freischalten?
 209=Bist du sicher?
 210=Trupp auswählen
 211=Trupp verlassen
 212=@1 wird bereits verwendet.
 213=@1 ist verfügbar.
-214=@1 wird bereits verwendet!
+214=@1 wurde die Rolle zugewiesen.
 215=@1 wurde noch nicht freigeschaltet.
 216=Du bist bereits ein @1.
 217=Als Sturmsoldat bist du an der Front des Schlachtfeld und greifst die Feinde frontal an.//Beschreibung des Sturmsoldats
-218=Als Sanitäter bist du dafür verantwortlich, verbündete Truppen zu heilen und am Leben zu halten.//Beschreibung des Sanitäters
-219=Du unterdrückst Feinde mit LMG`s, um deinen Team den Rücken freizuhalten.//Beschreibung des Unterstützer
-220=Als Unterstützer bist du spezialisiert, verbündete Fahrzeuge zu reparieren und feindliche Fahrzeuge zu zerstören.//Beschreibung des Ingenieurs
+218=Als Sanitäter bist du dafür verantwortlich, verbündete Truppen zu verarzten und am Leben zu halten.//Beschreibung des Sanitäters
+219=Als Unterstützer benutzt du dein LMG um deinem Team die Chance zu geben vorzurücken. // Beschreibung des Unterstützers
+220=Als Ingenieur bist du auf die Reparatur alliierter Fahrzeuge und die Zerstörung feindlicher Fahrzeuge spezialisiert.//Beschreibung des Ingenieurs
 221=Als Scharfschütze überwachst du das Schlachtfeld und schaltest gezielt und präzise Feinde in der Entfernung aus.//Beschreibung des Aufklärers
 222=@1 Punkte des Trupps
 223=@1 Plätze vorhanden//Klassen im Trupp verfügbar
-224=Randverziehung
+224=Vignette
 225=Maximale FPS
 226=Maus Sensitivität
 227=Ziel Sensitivität
-228=Schuss Sensitivität
-229=Rauch Granate
+228=Geschütz Sensitivität
+229=Rauchgranate
 230=@1 Basis
 231=@1 Ticket
 232=Wir haben @1 eingenommen
 233=Feind hat @1 eingenommen
 234=Neuer Sammelpunkt wurde gesetzt
 235=Du kannst in @1 Sekunden einen neuen Sammelpunkt setzen // Sammelpunkt cooldown
-236=Du brauchst mindestens 2 Teammitglieder in deiner nähe.
-237=Feinde sind in deiner nähe (50 Meter)
+236=Du brauchst mindestens 2 Teammitglieder in deiner Nähe.
+237=Feinde sind in deiner Nähe (50 Meter)
 238=Fernglas
 239=Claymore
-240=Drücke '@1' um @2 magazine/n(@3 Patronen vorhanden) aufzuheben
+240=Drücke '@1' um @2 Magazine/n(@3 Patronen vorhanden) aufzuheben
 241=Halte '@1' um den Abzug zu drücken
-242=Drücke '@1' um Detonieren zu lassen
+242=Drücke '@1' um detonieren zu lassen

```

Comment:
Sollte dies der String "Press @1 to detonate" würde ich "zum Detonieren" vorschlagen.

Reply:
changed to proposed